hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/Hip17UnhappyTokensSuite.java

```diff
@@ -241,9 +245,12 @@ private HapiSpec cannotAssociateNftWhenDeleted() {
                 .then(tokenAssociate(ANOTHER_USER, NFTdeleted).hasKnownStatus(TOKEN_WAS_DELETED));
     }
 
+    @HapiTest
     public HapiSpec cannotUpdateNftWhenDeleted() {
         return defaultHapiSpec("cannotUpdateNftWhenDeleted")
                 .given(
+                        fileUpdate(APP_PROPERTIES)
+                                .overridingProps(AutoRenewConfigChoices.propsForAccountAutoRenewOnWith(1, 0L, 2, 2)),
```

Comment:
Is this change necessary? How was it passing before on mono-service but not now without it? Is it a bug in mono service?

Reply:
Not 100% necessary, what this make is overrides the `bootstrap.properties` for `autoRenewDuration` as the test is trying to update a token with very low `autoRenewDuration`, which will fail in most cases only because by default in the `bootstrap.properties` file this is set to `ledger.autoRenewPeriod.minDuration=2592000`